tests/ui_tests/test_host_page.py

```diff
@@ -306,19 +311,20 @@ def test_delete_bonded_host(
     page.check_element_should_be_visible(HostListLocators.HostTable.row)
 
 
-@pytest.mark.full()
+@pytest.mark.smoke()
```

Comment:
Was full and now smoke? You have changed your mind about this test?

Reply:
It's "smoke". Was marked "full" by mistake originally